package.json

```diff
@@ -10,19 +10,15 @@
     "cdk": "cdk"
   },
   "dependencies": {
-    "aws-cdk": "^2.118.0",
-    "aws-cdk-lib": "^2.118.0",
+    "aws-cdk": "^2.162.1",
+    "aws-cdk-lib": "^2.162.1",
     "ts-node": "^10.9.2",
     "typescript": "~5.3.3"
   },
   "devDependencies": {
-    "@types/jest": "^29.5.11",
-    "constructs": "^10.3.0",
-    "jest": "^29.7.0",
```

Comment:
don't need jest anymore?

Reply:
In theory, we could have used jest to write unit tests against the infrastructure which is one of the nice things about codifying your environment in typescript.  The project is quite small, with little complexity and really never changes so I didn't think it was worth adding this the build.  Open to suggestions though.